index.bs

```diff
@@ -136,77 +137,170 @@ important to consider ways to mitigate the obvious impacts. For instance:
   How does this specification deal with sensitive information?
 </h3>
 
-Just because data is not personal information or PII, that does not mean
-that it is not sensitive information; moreover, whether any given information
-is sensitive may vary from user to user.  Data to consider if sensitive
-includes: financial data, credentials, health information, location, or
-credentials. When this data is exposed to the web, steps should be taken to
-mitigate the risk of exposing it.
-
-<p class=example>
-Credential Management [[CREDENTIAL-MANAGEMENT-1]] allows sites to request
-a user's credentials from a user agent's password manager in order to
-sign the user in quickly and easily. This opens the door for abuse, as
-a single XSS vulnerability could expose user data trivially to
-JavaScript. The Credential Management API mitigates
-the risk by offering the username and password as only an opaque
-{{FormData}} object which cannot be directly read by JavaScript
-and strongly suggests that authors use Content Security Policy [[CSP]]
-with reasonable [=connect-src=] and [=form-action=]
-values to further mitigate the risk of exfiltration.
-</p>
-
-<p class=example>
-Geolocation information can serve many use cases at a much less granular
-precision than the user agent can offer. For instance, a restaurant
-recommendation can be generated by asking for a user’s city-level
-location rather than a position accurate to the centimeter.
-</p>
-
-<p class=example>
-A Geofencing proposal [[GEOFENCING-EXPLAINED]] ties itself to service workers and
-therefore to encrypted and authenticated origins.
-</p>
+Personal information is not the only kind of sensitive information.
+Many other kinds of information may also be sensitive.
+What is or isn't sensitive information can vary
+from person to person
+or from place to place.
+Information that would be harmless if known about
+one person or group of people
+could be dangerous if known about
+another person or group.
+Information about a person
+that would be harmless in one country
+might be used in another country
+to detain, kidnap, or imprison them.
+
+Note:
+caste,
+citizenship,
+color,
+credentials,
+criminal record,
+demographic information,
+employment status,
+ethnicity,
+financial information,
+health information,
+location data,
+marital status,
+political beliefs,
+profession,
+race,
+religious beliefs or nonbeliefs,
+sexual preferences,
+and
+trans status
+are all examples of sensitive information.
+
+When a feature exposes sensitive information to the web,
+its designers must take steps
+to mitigate the risk of exposing the information.
+
+<div class=example>
+
+The Credential Management API allows sites
```

Comment:
This PR seems to mix stylistic rewordings like this paragraph, with changes in meaning like "or other party" in the first paragraph and "user"->"user agent" in the fingerprinting section. That makes it hard to review.

Reply:
Perhaps we could arrange time, either in a side meeting, or at a future meeting to discuss this? Personally I find document collaboration via GitHub to be more complex than other document collaboration tools such as those provided by Google or Microsoft.

In the meantime here is some background and related issues.

**PR Background** 

The PR is made against the "[revise-2](https://github.com/w3ctag/security-questionnaire/tree/revise-2)" branch of the master document. This PR introduces the following for reviewer consideration in addition to the changes already included in "revise-2".

1. Differentiate between "user" and "user agent". For example; a fingerprint can relate to a person, or a user agent. The harms, benefits and risks we are concerned about vary accordingly.

2. There are three different entities that might benefit or harm people’s privacy.

a)	First-parties – web site authors who are readily identifiable via the domain name displayed in the address bar.
b)	First-parties’ suppliers – organisations first-parties rely upon that are not identified in the address bar. 
c)	User agent vendors – organisations that make or control the web browser

People can be harmed by any one or all three of the above. Afterall some user agent vendors have received the largest fines in relation to privacy violations. 

The document would benefit from recognising each group and the role they play in the specific issues of security and privacy. This was originally raised as an [issue](https://github.com/w3ctag/security-questionnaire/issues/84) which includes some background.

**Related Issues**

These related issues are important. @torgo closed an issue related to [people's ability to trust supply chains](https://github.com/w3ctag/security-questionnaire/issues/83) with this statement “We have already established that the statement "supply chains can be trusted" (on its own) is false.” But unfortunately, did not provide me with any references to justify that that statement is false. Such information should be included in the document and be provided from an authoritative source. We can look at other industries they all have supply chains people can trust. Why should the web be different? As a minimum the document should be expanded to consider the conditions that do or do not enable a supply chain to be trust.

Referenced [RFC 6973](https://tools.ietf.org/html/rfc6973) entertains the possibility and certainly does not exclude it.

I believe the onus in a consensus driven governance model is for the proposer to convince others. As a new member I am keen to understand the facts and feel these issues have been closed without discussion or clarity in the document. I've been advised to progress resolution via text changes to the document and I'm doing so here.

**Policy**

The more general issue relates to W3C policy. This document infers policy. It does not seem to be the role of a standards body to make decisions that restrict people’s choice. That is a role of law makers.

Whilst these documents are supposed to advise they do prescribe rules akin to laws. According to the [W3C Process](https://w3c.github.io/w3process/index.html) if a proposal is to be successful it will at some point need to pass horizontal review. It's unlikely the AC or the Director would approve a standard that were deficient.

Rightly proposers become familiar with those documents that will be used to assess their proposal at the onset. Reviewer's frequently reference these documents. Whilst these documents may not be formally classified as laws, they appear to be use as such in practice.